tests/unit/sys/test_expressionparser.cxx

```diff
@@ -23,12 +23,8 @@ class ExpressionParserTest : public ::testing::Test {
   std::vector<double> t_array = {-1., 0., 1., 5., 10., 3.14e8};
 };
 
-Position LegacyPosition(BoutReal x, BoutReal y, BoutReal z, BoutReal t){
-  Position pos{0,0,0, CELL_CENTRE, nullptr, t};
-  pos.setX(x);
-  pos.setY(y);
-  pos.setZ(z);
-  return pos;
+Context LegacyPosition(BoutReal x, BoutReal y, BoutReal z, BoutReal t) {
```

Comment:
Yes probably, thanks! Once the name is settled this can be modified. I'm not sure about testing legacy uses of the code though, and would prefer to test ways that the code is intended to be used.